resteasy-reactive-context/src/test/java/org/resteasy/reactive/context/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package org.resteasy.reactive.context;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+    extends TestCase
+{
+    /**
+     * Create the test case
+     *
+     * @param testName name of the test case
+     */
+    public AppTest( String testName )
+    {
+        super( testName );
+    }
+
+    /**
+     * @return the suite of tests being tested
+     */
+    public static Test suite()
+    {
+        return new TestSuite( AppTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)
+     */
+    public void testApp()
+    {
+        assertTrue( true );
```

Comment:
It would be better to remove this at all :)

Or even better, write resteasy context test to testsuite module.

Or even better, write resteasy && cdi context test to testsuite module.

Reply:
Ah yes, that's a mistake (added by my IDE). Sorry, I'll remove it.